lib/fauna/objects.rb

```diff
@@ -0,0 +1,111 @@
+module Fauna
+  ##
+  # A Ref.
+  #
+  # Reference: {FaunaDB Special Types}[https://faunadb.com/documentation#queries-values-special_types]
+  class Ref
+    # The raw ref string.
+    attr_accessor :ref
```

Comment:
`str_value` seems verbose though, and we don't have any other type of value here.